CMakeLists.txt

```diff
@@ -209,6 +209,12 @@ install(FILES "${CMAKE_SOURCE_DIR}/Contributors.md"
 install(FILES "docs/scap-security-guide.8"
     DESTINATION "${CMAKE_INSTALL_MANDIR}/man8")
 
+if(SSG_OVAL_511_ENABLED)
+    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}")
+else()
+    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}-oval-5.10")
+endif()
+
```

Comment:
@redhatrises Are you talking about semantics of zipfile and tarball, or the technical approach?

Maybe I should also add the comment present in current `Makefile`
```
@# ZIP only contains source datastreams and kickstarts, people who
@# want sources to build from should get the tarball instead.
```